src/emscripten-optimizer/istring.h

```diff
@@ -29,6 +29,8 @@
 #include <stdio.h>
 #include <assert.h>
 
+#include "../support/threads.h"
```

Comment:
Weird include, doesn't `"support/threads.h"` work? We should just add it to the cmake search path if not.